image/spec/README.md

```diff
@@ -23,6 +23,15 @@ and in [OCI Image Format Specification](https://github.com/opencontainers/image-
 
 All 1.X versions are compatible with older ones.
 
+### [v1.3](https://github.com/moby/moby/blob/ff46faa238f7ea7e8deddfde6c46cdde07fd680c/image/spec/v1.3.md)
```

Comment:
This could point to the `spec.md` at the new commit (or "current"), but perhaps we should just remove the link.